_posts/2023-03-09-group-equivariant-cnn-1/group-equivariant-cnn-1.Rmd

```diff
@@ -0,0 +1,148 @@
+---
+title: "Upside-down, a cat's still a cat: Evolving image recognition with Geometric Deep Learning"
```

Comment:
```suggestion
title: "Upside down, a cat's still a cat: Evolving image recognition with Geometric Deep Learning"
```

Reply:
Only gets a hyphen if it's directly modifying a noun (e.g. "an upside-down cat")

Reply:
oh, right, thx!